### ISSUE TITLE
Should we remove the dose when dose has been applied...

### ISSUE THREAD

**Issue by pconesa** [scipion-em/scipion-em-tomo#297]
If we do dose weighted motion correction or if we do an imod dose filtering.....does is make sense to clear the dose information?

**Comment by azazellochg** [comment]
Not sure why would you do this? It's acquisition information. E.g. we dont remove tilt axis angle for aligned ts.

**Comment by pconesa** [comment]
If metadata describes the data, to me it make sense

**Comment by azazellochg** [comment]
True, then can we remove the tilt axis angle from an aligned ts? Should we
remove ctf from ctf-corrected ts? For me, acquisition metadata should never
be changed because it is related more to the experiment rather than the
data itself.
I don't see any problem of keeping the dose, ctf etc. Now we have flags for
ctf corrected/aligned data. You can add one more if you like, though I
think it's not necessary (e.g. in SPA there's no such problem of keeping
the track what was dose weighted or not)

Best regards,
Grigory

--------------------------------------------------------------------------------
Grigory Sharov, Ph.D.

MRC Laboratory of Molecular Biology,
Francis Crick Avenue,
Cambridge Biomedical Campus,
Cambridge CB2 0QH, UK.
tel. +44 (0) 1223 267228 <+44%201223%20267228>
e-mail: ***@***.***

**Comment by pconesa** [comment]
Thanks Grigory, this is why I was asking...

We can use flags to assist the user.

**Comment by azazellochg** [comment]
I have to correct myself, removing (setting to 0) tilt axis for aligned TS is very important. Most imod-inspired software expects the tilt axis to be vertical (=0 in imod) before reconstruction. But this is not relevant for the issue here.

**Comment by pconesa** [comment]
Well, I think metadata should describe the data is associated with. A different question is if Acquisition is important to remain during the whole process. For example, once a tilt series is interpolated the transformation matrix is removed, same with the tilt axis you mentioned.
If dose has been "applied" ( in the same way the transformation or the tilt axis has been applied) then I think it should go away so latter protocols do not apply the dose again. Now, if the does has to remain anyway, we can use a flag (doseApplied?)

**Comment by azazellochg** [comment]
dose is used in polishing again

**Comment by pconesa** [comment]
yes, but for polishing you go back to the movies, right?

**Comment by azazellochg** [comment]
True. I've never used relion for tomo, so I dont know how it works. @JorMaister ?
